scripts/yast2

```diff
@@ -35,6 +35,10 @@ export QT_IM_MODULE=xim GTK_IM_MODULE=xim
 # used correctly (see bnc#866692 for details).
 export PX_MODULE_PATH=""
 
+# Force using Perl readline library instead of the broken default Gnu variant
+# to avoid ugly error message printed in command line mode (bsc#966042)
+export PERL_RL=Perl
```

Comment:
Aghh, good point!

Debugging revealed that it's not installed by default and `Term::ReadLine::Stub` is silently used instead. That means there is actually no readline used - the command line works but there is no command history in the interactive mode. 

So there are these possibilities:

- Keep the broken default as it is (drop this patch)
- Modify the patch to keep the user defined `$PERL_RL` value (set it only if not defined). Users who do not like the new default (and want the history) could either override it to `Gnu` value back or install the `Term::ReadLine::Perl` library manually.

We should probably discuss it with @mvidner next week...